config/alphagov.yml

```diff
@@ -40,26 +40,10 @@ di-ipv-orange-cri-maintainers:
   <<: *common_properties
   repos:
     - di-ipv-acceptance-tests
-    - di-ipv-config
-    - di-ipv-cri-address-api
-    - di-ipv-cri-address-front
     - di-ipv-cri-check-hmrc-api
     - di-ipv-cri-check-hmrc-front
-    - di-ipv-cri-common-express
-    - di-ipv-cri-common-infrastructure
-    - di-ipv-cri-common-lambdas
-    - di-ipv-cri-dev-tools
-    - di-ipv-cri-kbv-api
-    - di-ipv-cri-kbv-front
-    - di-ipv-cri-kbv-hmrc-api
     - di-ipv-cri-kbv-hmrc-front
-    - di-ipv-cri-lib
-    - di-ipv-cri-otg-hmrc
-    - di-ipv-cri-pipeline-deployment
-    - di-ipv-cri-tech-docs
     - di-ipv-cri-templates
```

Comment:
All of these except `di-ipv-acceptance-tests` look like they've already moved to the other org